R/concRespCore.R

```diff
@@ -85,7 +85,7 @@ concRespCore <- function(row,
   )
 
   # calculate the hitcall
-  summary <- tcplhit2_core(params, conc, resp, cutoff, onesd, bmr_scale, bmed, conthits, aicc, identifiers,bmd_low_bnd, bmd_up_bnd)
+  summary <- tcplhit2_core(params, conc, resp, cutoff, onesd, bmr_scale, bmed, conthits, aicc, identifiers,bmd_low_bnd, bmd_up_bnd, errfun)
```

Comment:
Note, this one will revert back based on comment in tcplhit2_core of extracting the errfun from the fit param list output.